lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.t

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env genome-perl
+
+# P R A G M A S ###############################################################
+use strict;
+use warnings;
+
+# M O D U L E S ###############################################################
+use above 'Genome';
+use Path::Class;
+use Test::More;
+
+# M A I N #####################################################################
+
+#This is a bare minimum test that just compiles Perl and the UR class.
+use_ok('Genome::Model::Tools::Transcriptome::ErccMapUnaligned');
+done_testing();
+
+
+# This is an integration test that relies on data in production
```

Comment:
That was a decent reason.  I think you can use a [SKIP][] block instead.  I do recall that either the JUnit formatter for `prove` or the TAP parser for Jenkins (which we don't use) may have had an issue with `SKIP` or `TODO` so if you use that and have issue be aware.

[SKIP]: https://metacpan.org/pod/Test::More#SKIP:-BLOCK